docs/changelog.md

```diff
@@ -55,6 +55,15 @@ permalink: /changelog/
 * **feature-prompts**
   * Adds support for Web Share API using `ShareDelegate`.
 
+* **lib-dataprotect**
+  * Added new `SecureAbove22Preferences` helper class, which is an encryption-aware wrapper for `SharedPreferences`. Only actually encrypts stored values when running on API23+.
+
+* **service-firefox-accounts**
+  * Support for keeping `SyncEngine.Passwords` engine unlocked during sync.
+
+* **concept-sync**
+  * Added new `LockableStore` to facilitate syncing of "lockable" stores (such as `SyncableLoginsStore`).
+
```

Comment:
Do I understand this correctly that this is not breaking anything existing (e.g. Lockwise will just continue to work without any migration)? :)

Reply:
Yeah, not a breaking change.